extensions/amp-live-list/0.1/amp-live-list.js

```diff
@@ -54,6 +54,7 @@ export class LiveListInterface {
    * Update the underlying live list dom structure.
    *
    * @param {?Element} element
+   * @return {time}
```

Comment:
We do have a typedef and I usually ask everyone to use it. Although, if we decide it's not useful, let's remove.